recipes/devito-icc-mpich/shipyard-config/jobs.yaml

```diff
@@ -0,0 +1,26 @@
+job_specifications:
+- id: bench-icc-mpich
+  shared_data_volumes:
+  - azureblob_vol
+  tasks:
+  - docker_image: gbisbas/devitoicc:v0.3
+    environment_variables:
+      DEVITO_ARCH: intel
+      OMP_NUM_THREADS: 8
+      OMP_PROC_BIND: 'close'
+      DEVITO_MPI: 1
+      DEVITO_OPENMP: 1
+      OMP_PLACES: 'cores'
+      DEVITO_LOGGING: 'DEBUG'
+      LC_ALL: 'C.UTF-8'
+      LANG: 'C.UTF-8'
+    default_working_dir: container
+    multi_instance:
+      num_instances: pool_current_dedicated
+      mpi:
+        runtime: mpich
+        processes_per_node: 2
+        options:
+          - --bind-to socket
+      pre_execution_command: source /opt/intel/compilers_and_libraries/linux/bin/compilervars.sh -arch intel64 -platform linux; source /opt/intel/compilers_and_libraries/linux/mpi/intel64/bin/mpivars.sh; pip3 install mpi4py; 
```

Comment:
why are you sourcing intel mpi for mpich run?

Reply:
remianders